src/main/java/org/rabbitconverter/rabbit/Rabbit.java

```diff
@@ -39,6 +39,8 @@
    */
   public static String uni2zg(String input) {
 
+    if (input == null) return null;
```

Comment:
If should be on Rabbit main repo instead of this because Rabbit.java file is generate by Rabbit Template engine.